subiquity/curtin.py

```diff
@@ -91,7 +94,7 @@ def curtin_userinfo_to_config(userinfo):
         '    groups: admin\\n' + \
         '    lock-passwd: false\\n'
     if 'ssh_import_id' in userinfo:
-        user_template += '    ssh-import-id: [{ssh_import_id}]\\n'
+        user_template += '    ssh-import-id: [ "{ssh_import_id}" ]\\n'
```

Comment:
BTW, I have this fixed properly in https://github.com/CanonicalLtd/subiquity/pull/211. The reason this fix doesn't help is the levels of nesting -- this is cloud-init config, which is yaml that gets embedded as a string inside the yaml of the curtin config. Drop this bit?